homeassistant/components/wilight/strings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "config": {
+    "flow_title": "WiLight: {name}",
+    "step": {
+      "confirm": {
+        "title": "WiLight",
+        "description": "Do you want to set up WiLight {name}?\n\n It supports: {components}"
+      }
+    },
+    "abort": {
+      "already_configured": "This WiLight is already configured",
```

Comment:
Please use a reference as I linked before.

Reply:
I tried to change "already_configured" to "already_configured_device" but when I ran test_config_flow.py / test_ssdp_device_exists_abort, it gave an error because "already_configured" is created automatically by _abort_if_unique_id_configured() used in config_flow..